migrations/1618507684019-rename-0-conf.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash/fp')
+const settingsLoader = require('../lib/new-settings-loader')
+const configManager = require('../lib/new-config-manager')
+
+const isNil = val => val == null
```

Comment:
You have this function available in lodash. `_.isNil()` if you want to check for `null` and `undefined`, `_.isNull()` if you want to check for `null` only